olive/passes/onnx/perf_tuning.py

```diff
@@ -56,6 +56,14 @@ def tune_onnx_model(model, config):
     latency_user_config = {}
     # which should be the same as the config in the metric
     config_dict = config.dict()
+
+    # use model io_config if user does not specify input_names and input_shapes
+    if not config_dict.get("input_names") or not config_dict.get("input_shapes"):
+        io_config = model.get_io_config()
```

Comment:
Should we just remove the input_names and input_shapes config options from perf_tuning pass?